Description: sync pbr version between requirements.txt and setup.py
Author: Michael Fladischer <fladi@debian.org>
Last-Update: 2015-07-31
Forwarded: no

Index: python-mock/setup.py
===================================================================
--- python-mock.orig/setup.py	2015-07-15 02:06:46.000000000 +0200
+++ python-mock/setup.py	2015-07-31 18:37:58.533472306 +0200
@@ -14,5 +14,5 @@
     sys.exit(1)
 
 setuptools.setup(
-    setup_requires=['pbr>=1.3'],
+    setup_requires=['pbr>=0.11'],
     pbr=True)
